src/main/scala/com/softwaremill/react/kafka/KafkaActorSubscriber.scala

```diff
@@ -23,17 +22,12 @@ private[kafka] class KafkaActorSubscriber[T](
       processElement(element.asInstanceOf[T])
     case ActorSubscriberMessage.OnError(ex) =>
       handleError(ex)
-    case ActorSubscriberMessage.OnComplete =>
+    case ActorSubscriberMessage.OnComplete | "Stop" =>
```

Comment:
To be honest: for tests only. I know that's pretty smelly but I couldn't find a better way to implement an important test (`ReactiveKafkaIntegrationSpec. "close producer, kill actor and log error when in trouble"`). Using `TestActorRef.underlyingActor` couldn't work there because I already had an instance of `ActorRef` which needs to be supervised. Sending `ActorSubscriberMessage.OnComplete` triggers some internal akka stuff that I didn't want to get executed.